Clean-up and some functions added

niklasfuks opened the pull request:
whats done:
Cannot edit name, removed email addres, removed manage social accounts,
removed manage social accounts, added a profile page to se your posts,
removed everything with password and sign up forms.
Cleaned the front page

added positions finder on front page

Added Dynamicly generation of page to a templete.

Commit:
Clean-up and some functions added

whats done:
Cannot edit name, removed email addres, removed manage social accounts,
removed manage social accounts, added a profile page to se your posts,
removed everything with password and sign up forms.
Cleaned the front page

added positions finder on front page

Added Dynamicly generation of page to a templete.

timgranstrom committed:
Added missing semicolons for routes

Fixed bug where semicolons was missing in the configs for core client
and users client routes.

timgranstrom commented:
Signin and singup view is practically the same. Since you can only signup using facebook, there is no need to have the actual signup view. (Loggining in using facebook oauth will automatically register the user if they don't exist).

Remove singup.client.view.html view and all it's dependencies!

---

Since we can only authenticate using oauth2 there is no need to use the "signin.client.view.html" view, since it is only a signin view for "local account" that isn't registered using oauth2.

Remove the signin.client.view.html view and all it's dependencies!
Edit the authentication.client.view.html view to say "Sign in or sign up using your facebook account" rather than just "Sign in using your facebook account", since you can actually register through this link as well.

Commit:
Fixed due to comments

Remove singup.client.view.html nad signin.client.view.html view and all
it's dependencies.

Edit the authentication.client.view.html view to say "Sign in or sign up
using your facebook account"

Commit:
Disabled changeble user form

First name and last name are not changable more in
edit-profile.client.view.html
Fixed tag error in user-edit.clent.view.html
